.github/workflows/style.yml

```diff
@@ -19,16 +15,21 @@ jobs:
     name: Style Checker
     runs-on: ubuntu-latest
     steps:
+      # Unfortunately there doesn't seem to be a way to
+      # do this without an extra step.
+      - name: Get depth
+        run: |
+          echo "DEPTH=$(( ${{ github.event.pull_request.commits }} + 1 ))" > $GITHUB_ENV
```

Comment:
I guess `$GITHUB_ENV` is magic and might be worth a reference in the commit message.

Reply:
And it should be `>>` in order not to overwrite it (even if it doesn't matter if it's overwritten).
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-an-environment-variable

Reply:
> I guess `$GITHUB_ENV` is magic and might be worth a reference in the commit message.

I added it to the commit message, let me know if it's too vague. :)